Lib/pdb.py

```diff
@@ -925,14 +925,22 @@ def do_ignore(self, arg):
         condition evaluates to true.
         """
         args = arg.split()
-        try:
-            count = int(args[1].strip())
-        except:
+        if not args:
+            self.error('Breakpoint number expected')
+            return
+        if len(args) == 1:
             count = 0
+        elif len(args) == 2:
+            try:
+                count = int(args[1])
```

Comment:
The current code uses `args[1].strip()`. 
Even though I don't have an example for when this would be useful, I suggest not to omit `.strip()` here to maintain consistency with the rest of the code here.

Reply:
`args[1].strip()` only strips spaces, which `int()` takes care of when it tries to convert the string to integer. This is redundant so I removed it.

Reply:
Fair enough, thanks for explaining